plugins/outputs/riemann/riemann.go

```diff
@@ -68,8 +79,16 @@ func (r *Riemann) Write(metrics []telegraf.Metric) error {
 
 	var senderr = r.client.SendMulti(events)
 	if senderr != nil {
```

Comment:
Yes. But will Telegraf try to re-attempt to send metrics again? Or they would be dropped if send fails?

Anyway, I removed immediate reconnection attempt on send failure.